AIDevGallery/Utils/AppUtils.cs

```diff
@@ -158,4 +161,32 @@ public static ImageSource GetModelSourceImageFromUrl(string url)
             return new SvgImageSource(new Uri("ms-appx:///Assets/ModelIcons/HuggingFace.svg"));
         }
     }
+
+    public static StyleDictionary GetCodeHighlightingStyleFromElementTheme(ElementTheme theme)
```

Comment:
This method is not available to exported projects. That is why the tests are failing. We need another solution here.
Either making this method available for the exported projects, or not calling this method on the exported projects (with `// <exclude>` maybe?).

Reply:
I'll look into it.